src/components/card.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Map from './map';
+import Quality from './quality';
+
+const Card = ({ data }) => (
+  <li className="card">
+    <Map className="card-img" lati={data.lat} lon={data.lon} />
+    <div className="card-img-overlay">
+      <h5 className="card-title">{`${data.name},  ${data.state}, ${data.country}`}</h5>
+    </div>
+    <Quality className="card-text card-footer" number={data.aqi} />
+  </li>
+);
```

Comment:
- [ ] There is an issue with your design. one of the requirement of this project is to have a unique detail route for every item on your homepage. so when the user clicks on the title of any item on your homepage, it would take him to the detail page of the item. you can use `useNavigate` or `useParams` from react-router-dom for that purpose.
so please consider implementing this requriement.
-  Home page
  -  When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for 
  example by the category name.
  -  Along with the category name, you will display some numeric values per category that come from the API.
  -  When the user clicks (or taps) on a category item, the application navigates to the details page.

Reply:
Hi @Zuhaib042 thanks for the feedback, I have some questions, should I remove the current home page 
![image](https://github.com/PabloEGonz/air-quality-app/assets/68977631/6a07053d-3527-4e80-b8ad-bde56fe53e87)
and place the filter page instead? 
![image](https://github.com/PabloEGonz/air-quality-app/assets/68977631/bb93196c-363a-4cf6-83f6-3e6d9d24ac81)